bindings/py/dummy.c

```diff
@@ -1,9 +0,0 @@
-#include <stdio.h>
```

Comment:
so I understand the removed `bindings/py/` is replaced with the PyBind functionality? 
You've mentioned replacing the PyRegion with something else..?

Reply:
Correct

Reply:
Even with the tests? You could run `REPO_DIR$ py.test bindings/py/`

Reply:
I need to update to python 3 first. Hold on